src/YoloDev.Expecto.TestSdk/build/net6.0/YoloDev.Expecto.TestSdk.props

```diff
@@ -1,10 +1,26 @@
 <Project>
+
+  <PropertyGroup>
+    <!-- Makes the new platform/runner opt-in feature -->
+    <EnableExpectoRunner Condition=" '$(EnableExpectoRunner)' == '' ">false</EnableExpectoRunner>
```

Comment:
Now that I come back to this PR after a while away, I think this property probably needs to change name. "EnableExpectoRunner" doesn't tell me (and probably even less anyone else) anything. How about `EnableExpectoTestingPlatformIntegration` - it's a bit verbose, but I think explicit is better. It would also be quite good if a property that all the test frameworks could agree on would be named (like just `EnableTestingPlatform`) that we could default to - but we can leave that for later.

Reply:
Reading your comment and given we might end up doing tools to migrate people away from VSTest, I have raised the point for discussion with teams and PMs so we could settle on some common naming. At the moment, NUnit and xUnit are only in previews/betas for the support of our new platform so this is still a good time for doing the change. We would have 2 properties to support on MSTest for some time but that's not too bad.

Reply:
Please let me know if anything comes from this